content/about.md

```diff
@@ -28,49 +28,56 @@ cover:
 
 ### Work-related interests:
 
-- Currently designing a commercial cubesat platform at [SatRevolution](https://satrevolution.com/)
-- Playing around with UAVs - MavLink, ArduPilot & DroneKit
-- Vice president of the [Academic Aviation Club](http://akl.pwr.edu.pl)
-- Writing automation & testing scripts mainly in Python & Bash
+- Currently designing & operating a commercial CubeSat constellation @[SatRevolution](https://satrevolution.com/)
+- Playing around with UAVs - ArduPilot, computer vision, telemetry systems
+- Tech lead of the [Academic Aviation Club](http://akl.pwr.edu.pl) (2nd place [Sae Aero Design Advanced](https://www.saeaerodesign.com/) winner)
 - Extensive Linux knowledge (long time power user, ricing enthusiast)
-- Solid understanding of C++ and low-level, planning to learn Rust
-- Web services and web apps with Flask and Django
+- Solid understanding of mission operations & telemetry system design
+- Python + Rust + Bash = ❤️
 
 ### Other interests:
 
 - Hard sci-fi
-- Lots of music, weird sounds
-- Random philosophical ideas
+- Overly pretentious music
+- Messing around with synthesizers
+- Philosophy
 
 ---
 
-## UAVs
+## CubeSats
 
-Since 2019, I've been rolling with the [Academic Aviation Club](http://akl.pwr.edu.pl) - a student club from my uni,
-dedicated to developing autonomous airplanes and drones. I've got the know-how to develop on top of ArduPilot-based flight controllers.
+During the summer of 2020 I've started working at [SatRevolution](https://satrevolution.com/) - company
+designing a cubesat satellite fleet for commercial and research usage.
 
-As a confimation of my skills: I've attended [SAE Aero Design 2020](https://www.saeaerodesign.com/) and my team has placed second in the Advance cathegory.
+I currently lead the mission operations software team -- that's where we design
+the communication & telemetry systems of our satellites, create the tools used for automated
+testing and finally talk with the satellites on orbit.
```

Comment:
hmm create the tools used for automated testing, by this I understand that u create software for automation, not that you are using them to automate your tests?
'talk to' sounds better

Reply:
This is kinda complicated, as the core idea resolves mostly around QA/smoke testing of the satellites themselves. Maybe `automated satellite testing` will get the message through better?

Reply:
let it be